app/uk/gov/hmrc/securemessage/controllers/models/generic/ConversationMetadata.scala

```diff
@@ -69,6 +69,9 @@ object ConversationMetadata {
   private def findLatestMessage(coreConversation: Conversation): Message =
     coreConversation.messages.sortBy(_.created.getMillis).reverse.head
 
+  private def isLatestMessageBySender(coreConversation: Conversation, participant: Participant) =
+    coreConversation.messages.sortBy(_.created.getMillis).reverse.head.senderId === participant.id
```

Comment:
It's dangerous to call `.head` here. Can we replace it with `.headOption`

Reply:
@alexflav23 messages is Cats nonempty do you think its still dangerous ?

Reply:
ah ok that's great!